frontend/src/routes.tsx

```diff
@@ -1,14 +1,18 @@
-import React from 'react';
-import { Router } from '@reach/router';
+import React from "react";
+import { Router } from "@reach/router";
 
 import { Layout } from './layout/layout.component';
 import { DashboardPage } from './pages/dashboard.page';
+import { HomePage } from "./pages/home.pages";
 
 export function Routes() {
   return (
     <Layout>
       <Router>
-        <DashboardPage/>
+        <HomePage path="/home"/>

```

Comment:
Let's add the default prop to the route:

![image](https://user-images.githubusercontent.com/25971028/173173969-80be1d2f-b3d4-4d01-9792-64d59ff82bfa.png)


Please, take a look at the documentation for Reach Router: https://reach.tech/router/tutorial/08-default-routes